proto/buf/alpha/registry/v1alpha1/resolve.proto

```diff
@@ -33,10 +33,24 @@ service ResolveService {
   rpc GetModulePins(GetModulePinsRequest) returns (GetModulePinsResponse) {
     option idempotency_level = NO_SIDE_EFFECTS;
   }
-
-  // GetRemotePackageVersion resolves the given plugin and module references to a version.
+  // GetGoVersion resolves the given plugin and module references to a version.
+  // The format of the version is determined by the plugin type.
```

Comment:
```suggestion
```

I think you can nix the second line for each of these, as the format of the version is fixed per-registry.